### ISSUE TITLE
affliate sytem

### ISSUE THREAD

**Issue by obaviet** [spurtcommerce/spurtcommerce#6]
Hello
I'm new to Spurs and really impressed with everything. It's great for an open source platform when the backend and fontend are both very nice, which other open source code doesn't have (they make very bad fontend). Thanks to the developers
However it seems that we do not have a strong marketing system including affiliate marketing, even that there is an advertising system.
Hope these 2 features will be of interest

**Comment by spurtcomm** [comment]
> Hello I'm new to Spurs and really impressed with everything. It's great for an open source platform when the backend and fontend are both very nice, which other open source code doesn't have (they make very bad fontend). Thanks to the developers However it seems that we do not have a strong marketing system including affiliate marketing, even that there is an advertising system. Hope these 2 features will be of interest

Yes, we are currently planning integration of third party tools for affiliate marketing, remarketing and retargeting. Soon, we will give you the good news, once it is implemented. Thank you for your interest !
